src/cheri/riscv-unpriv-integration.adoc

```diff
@@ -253,7 +253,7 @@ image::../cheri/img/cap-bounds-map.png[width=80%,align=center]
 ==== Memory space
 
 A hart supporting {cheri_base_ext_name} has a single byte-addressable address
-space of 2^XLEN^ bytes for all memory accesses. Each memory region capable of
+space of 2^MXLEN^ bytes for all memory accesses. Each memory region capable of
```

Comment:
I thought we dropped this on purpose because we're in the unpriv section and can't talk about MXLEN?

Reply:
ahhh but the paragraph is inconsistent, I'll make them all XLEN